src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitations/GetInvitationsQueryHandlerTests.cs

```diff
@@ -0,0 +1,582 @@
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 2);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnPunchOutInNextWeek()
+        {
+            var filter = new Filter { PunchOutDates = new List<PunchOutDateFilterType> { PunchOutDateFilterType.NextWeek } };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data,0);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnLastChangedAtFrom()
+        {
+            var filter = new Filter { LastChangedAtFromUtc = new DateTime(2019, 10, 20, 10, 0, 0, DateTimeKind.Utc) };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 2);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnLastChangedAtFrom2()
+        {
+            var filter = new Filter { LastChangedAtFromUtc = new DateTime(2020, 10, 20, 10, 0, 0, DateTimeKind.Utc) };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 0);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnLastChangedAtTo()
+        {
+            var filter = new Filter { LastChangedAtToUtc = new DateTime(2019, 10, 20, 10, 0, 0, DateTimeKind.Utc) };
+
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(_projectName2, null, filter);
+                var dut = new GetInvitationsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+                AssertCount(result.Data, 0);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handler_ShouldFilterOnLastChangedAtTo2()
```

Comment:
rename Handler_ShouldFilterOnLastChangedAtTo2 so reader understand why expecting result is as it is